### ISSUE TITLE
java.lang.UnsatisfiedLinkError: dlopen failed: library "libskiko-android-arm64.so" not found

### ISSUE THREAD

**Issue by hafiz013** [JetBrains/skiko#970]
I try this following below code but got crash above:

```
@OptIn(ExperimentalEncodingApi::class)
    private fun ByteArray.toBase64(): String {
        val image: Image = makeFromEncoded(this)
        val bitmap: Bitmap = image.asSkiaBitmap()
        val encodedBytes = Image.makeFromBitmap(bitmap).encodeToData(EncodedImageFormat.PNG, 60)?.bytes
        return encodedBytes?.let {
            Base64.Default.encode(it)
        } ?: run {
            ""
        }
    }

    private fun Image.asSkiaBitmap(): Bitmap {
        val bitmap = Bitmap()
        this.readPixels(bitmap)
        return bitmap
    }
```

this is my import library:

```
commonMain.dependencies {
    implementation(libs.skiko)
}

......
skiko = "0.8.4"
skiko = { module = "org.jetbrains.skiko:skiko", version.ref = "skiko" }
```

How to solve this issue? My kotlin multiplatform is ios and android.

**Comment by kexuejin** [comment]
I have temporarily solved this problem, you can refer to #761

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/SKIKO-970">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
